aikau/src/main/resources/alfresco/forms/Form.js

```diff
@@ -309,7 +310,27 @@ define(["dojo/_base/declare",
        * @since 1.0.32
        */
       warningsPosition: "top",
-      
+
+      /**
+       * Should the first field in the form be focused when the form is loaded?
+       *
+       * @instance
+       * @type {boolean}
+       * @default
+       * @since 1.0.40
+       */
+      firstFieldFocusOnLoad: false,
```

Comment:
What happens if there is more than one form on the page and that both request firstFieldFocusOnLoad to be true? Or there is another widget that also requests focus. This configuration attribute is making a promise that is can't necessarily keep.